Cython/Compiler/ExprNodes.py

```diff
@@ -2135,7 +2135,7 @@ def analyse_as_type(self, env):
 
         # Try to give a helpful warning when users write plain C type names.
         if not env.in_c_type_context and PyrexTypes.parse_basic_type(self.name):
-            warning(self.pos, "Found C type '%s' in a Python annotation. Did you mean to use a Python type?" % self.name)
+            warning(self.pos, "Found C type '%s' in a Python annotation. Did you mean to use a 'cython.%s'?" % (self.name, self.name))
```

Comment:
Given that we say "Found C type …" and not "Found the C type …", I think we should be consistent and go without the article at the end as well.
```suggestion
            warning(self.pos, "Found C type '%s' in a Python annotation. Did you mean to use 'cython.%s'?" % (self.name, self.name))
```